tests/rhbz_functions.at

```diff
@@ -0,0 +1,275 @@
+# -*- Autotest -*-
+
+AT_BANNER([rhbz functions])
+
+## -------------- ##
+## rhbz_functions ##
+## -------------- ##
+
+AT_TESTFUN([rhbz_functions],
+[[
+
+#define PACKAGE_NAME "libreport"
+#define VERSION "1.0"
+
+#include "testsuite.h"
+#include "internal_libreport.h"
+#include "client.h"
+#include "abrt_xmlrpc.h"
+#include "rhbz.h"
+#include "proxies.c"
+#include "rhbz.c"
+#include "abrt_xmlrpc.c"
+#include <errno.h>
+#include <string.h>
+
+char *read_string(xmlrpc_env *env, xmlrpc_value *xml, const char *member)
+{
+        xmlrpc_value *xml_member = NULL;
+        char *ret = NULL;
+
+        xmlrpc_struct_find_value(env, xml, member, &xml_member);
+        if (env->fault_occurred)
+        {
+            printf("Error searching for member %s: %s\n", member, env->fault_string);
+            return ret;
+        }
+        xmlrpc_read_string(env, xml_member, (const char **)&ret);
+        if (env->fault_occurred)
+        {
+            printf("Error reading string from member %s: %s\n", member, env->fault_string);
+            return ret;
+        }
+        xmlrpc_DECREF(xml_member);
+
+        return ret;
+}
+
+TS_MAIN
+{
+    pid_t pid = fork();
+
+    if (pid == 0)
+    {
+        int ret = 0;
+        char *argv[2];
+        argv[0] = (char *)"../../mock_bugzilla_server/mock_bugzilla_server.py";
+        argv[1] = NULL;
```

Comment:
```suggestion
        char *argv[2] =
        {
            (char *)"../../mock_bugzilla_server/mock_bugzilla_server.py";
            NULL,
        };
```

Reply:
Though I would personally cast the array itself in the call to `execv()`.

Reply:
Rewritten, though casting in the call (instead of in the definition of `argv`) makes the compiler complain about a discarded `const` in the assignment.

Reply:
Well, the idea is that you make the array const (and you can drop the size, really). It’s only the `execv()` that is idiosyncratic.

Reply:
Ah, thanks. Rewritten.